component/identity/src/test/resources/conf/picketlink/exo.portal.component.identity-picketlink-idm-config.xml

```diff
@@ -30,6 +30,24 @@
       <identity-type-mappings>
         <user-mapping>USER</user-mapping>
       </identity-type-mappings>
+      <options>
+        <option>
+          <name>template</name>
+          <value>true</value>
+        </option>
+        <option>
+          <name>credentialEncoder.class</name>
+          <value>org.picketlink.idm.impl.credential.DatabaseReadingSaltEncoder</value>
+        </option>
+        <option>
+          <name>credentialEncoder.hashAlgorithm</name>
+          <value>SHA-256</value>
+        </option>
+        <option>
+          <name>credentialEncoder.secureRandomAlgorithm</name>
+          <value>SHA1PRNG</value>
+        </option>
+      </options>

```

Comment:
Should move this into `supported-identity-object-types`/`<identity-object-type>`/`<options>` of `HibernateStore`.
In fact, when declaring `HibernateStore` and other stores, like `LDAP`, this option will have to be defined for Hibernate only. BTW, please update other PLIDM config files in the same folder, section `HibernateStore`.

Reply:
In picketlink, the encoder is an option of the realm, not of the store.

I understand the concern, and share, but PL seems not allowing it

Reply:
> In picketlink, the encoder is an option of the realm, not of the store.

ok I see. In PLIDM Ldap Store source code, it seems not used. For my information, did you tested on LDAP conf ?

Reply:
I didnt test as LDAP connexion do not hash password

Reply:
I just test, and the login is ok